src/components/bookList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Book from './book';
+
+const BooksList = () => {
+  const bookList = [
+    {
+      id: 1,
+    },
+    {
+      id: 2,
+    },
+  ];
+  const list = bookList.map((item) => (<li key={item.id}><Book /></li>));
+  return (
```

Comment:
- [ ] Now that you've mapped through the bookList array, in the Book component, you can pass the single mapped item as a prop to the Book component so it will be received later in that component that you will be rendering. 

```js
const list = bookList.map((item) => (<li key={item.id}><Book item={item} /></li>));
```